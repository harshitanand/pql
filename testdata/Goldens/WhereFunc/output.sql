SELECT * FROM "Tokens" WHERE coalesce((abs("Value")) = (1), FALSE);
