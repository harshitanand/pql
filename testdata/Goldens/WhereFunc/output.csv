Kind,TokenConstant
1,TokenIdentifier
-1,TokenError
